Readme.md

```diff
@@ -1,4 +1,4 @@
-![GDevelop logo](https://raw.githubusercontent.com/4ian/GDevelop/master/newIDE/GDevelop%20banner.png "GDevelop logo")
+[![GDevelop logo](https://raw.githubusercontent.com/4ian/GDevelop/master/newIDE/GDevelop%20banner.png "GDevelop logo")](https://gdevelop.io)
```

Comment:
I think this image should be kept as is to avoid losing people visiting the repository.